semgrep/semgrep/semgrep_main.py

```diff
@@ -356,6 +357,8 @@ def main(
             for rule_match in rule_matches:
                 if rule_match._is_ignored:
                     num_findings_nosem += 1
+                    if sarif_include_ignores:
```

Comment:
Should we avoid the additional CLI arg and always add these?

Reply:
I'm supportive. It does break the assumption that people may have built around that all findings printed in the output are new findings but it does conform to sarif spec.

Reply:
I think it's no ideal for developers to see suppressed findings as it would be confusing but IMO developers should be shown the plain text output anyway and the sarif is for those who are building out some other tooling. @ShaylanDias mind modifying the PR so that behavior is to always include the suppressed finding in the sarif and for now not including an extra flag to remove it.

Reply:
Makes sense to me since it avoids more complications in the tool— but like you said @brendongo I didn't want to break any assumptions people might have already made. However, if there's consensus that the CLI arg is unnecessary I think it makes sense to remove it.